docs/installation.md

```diff
@@ -0,0 +1,22 @@
+### How can I install howdoi?
+
+To install howdoi the most straight forward method is through [pip](https://pip.pypa.io/en/stable/). If you already have pip installed you can simply run:
+
+``
+pip install howdoi
+``
+
+or
+
+``
+pip install git+https://github.com/gleitz/howdoi.git#egg=howdoi
+``
+
+!!! Note
```

Comment:
I would suggest a similar change to Note as I said above just to keep the consistency